core/common/src/test/java/alluxio/util/IdUtilsTest.java

```diff
@@ -30,4 +30,27 @@ public void getRandomNonNegativeLongTest() throws Exception {
     Assert.assertTrue(second >= 0);
     Assert.assertTrue(first != second);
   }
+
+  /**
+   * Tests if output of {@link IdUtils#createFileId(long)} is valid.
+   */
+  @Test
+  public void createFileIdTest() throws Exception {
+    long containerId = 1;
+    long fileId = IdUtils.createFileId(containerId);
+    Assert.assertTrue(fileId != -1);
```

Comment:
could you make it like this to be consistent with current Alluxio junit test style?
```java
Assert.assertNotEqual(-1, fileId)
```